exporters/committime/app.py

```diff
@@ -133,6 +162,9 @@ def make_collector(self) -> AbstractCommitCollector:
                 username=self.username,
                 token=self.token,
                 namespaces=self.namespaces,
+                app_label=self.app_label,
```

Comment:
I don't see the COMMIT_HASH_ANNOTATION nor COMMIT_REPO_URL_ANNOTATION for the *git* provider, only the *image* provider:

https://pelorus.readthedocs.io/en/latest/GettingStarted/configuration/ExporterCommittime/#provider-git-options

vs

https://pelorus.readthedocs.io/en/latest/GettingStarted/configuration/ExporterCommittime/#provider-image-options

Reply:
It is on the top level, along LOG_LEVEL and APP_LABEL https://pelorus.readthedocs.io/en/latest/GettingStarted/configuration/ExporterCommittime/#commit-time-exporter-configuration-options

I sit wrong in the documentation?

Reply:
Yes, the commit hash annotation and url annotation should be applied only to the image provider.

Reply:
going to change in the documentation then